widgetssdk/src/main/java/com/glia/widgets/chat/helper/FileHelper.java

```diff
@@ -55,6 +54,13 @@ public static String getFileName(AttachmentFile attachmentFile) {
     }
 
     public static String getFileName(String fileId, String fileName) {
-        return fileId + "." + fileName;
+        return fileId + "." + getFileExtension(fileName);
+    }
+
+    public static String getFileExtension(String fullName) {
```

Comment:
do files always have extensions?

Reply:
atm yes, but ill move dot to getFileExtensions method

Reply:
I am just thinking if they upload a file without dot/extension